org.gnome.DejaDup.yaml

```diff
@@ -0,0 +1,62 @@
+# -*- Mode: YAML; indent-tabs-mode: nil; tab-width: 2 -*-
+---
+app-id: org.gnome.DejaDup
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+command: deja-dup
+
+finish-args:
+  - --filesystem=host
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.gtk.vfs.*
+
+modules:
+  - ./fasteners.json
+  - ./future.json
+
+  - name: lib2to3  # needed for past module (from future module)
+    buildsystem: simple
+    build-commands:
+      - cp -r Lib/lib2to3 /app/lib/python3.7/site-packages/
+    sources:
+      - type: git
+        url: https://github.com/python/cpython
+        tag: v3.7.4
+        commit: e09359112e250268eca209355abeb17abf822486
+
+  - name: librsync
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/librsync/librsync
+        tag: v2.0.2
+        commit: 400ecbd28ac94d9a363c5ff7e58aea93cb1d90c4
+
+  - name: duplicity
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py build
+      - python3 setup.py install --prefix /app
+    sources:
+      - type: archive
+        url: https://launchpad.net/duplicity/0.8-series/0.8.05/+download/duplicity-0.8.05.tar.gz
+        sha256: 064672cf492003ff05f2d7ca8493d12af86de06aa1c60d77ac8dd74b2da3111d
+      - type: patch
+        path: ./python3.patch
+        strip-components: 0
+
+
+  - ./pydrive.json
+
+
+  - name: deja-dup
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/World/deja-dup
+        commit: 6b7129515d7575a3038c9acc56d95cadc7c51b1a
```

Comment:
Use a versioned `tag` if you can.

Reply:
Agreed. I'm just pointing at a follow-on hotfix for the appstream validation issue above.